scripts/oracle/mock_price_feed.py

```diff
@@ -4,7 +4,7 @@
 from pathlib import Path
 import sys
 
-CMD = str(Path.home()) + "/go/bin/seid"
+CMD = "printf '12345678\n' | " + str(Path.home()) + "/go/bin/seid"
```

Comment:
Can we leave a comment here about this is for internal testing only

Reply:
:+1: